### PR TITLE
added snow-extrapolation materials

### PR DESCRIPTION
**Pull request by whitelightning450**

### CODE REVIEW

**Comment by whitelightning450** [comment]
Testing book format, added tutorial placeholder and placeholder notebook.

**Comment by scottyhq** [comment]
Thanks @whitelightning450 , you can now see the preview of your notebook content here https://deploy-preview-56--geo-smart-book.netlify.app/tutorials/snow-extrapolation/introduction , go ahead and keep pushing commits until you're happy with the content

**Comment by whitelightning450** [comment]
Notes for website
Pip install pandas==1.4.3, h5py, tqdm, tables, scikit-learn, seaborn, tensorflow, progressbar, contextily, hydroeval, geopandas==0.10.2, folium==0.12.1.post1, vincent, hvplot==0.8.0, nbformat==5.7.0, pystac_client, planetary_computer, rioxarray, matplotlib, basemap, numpy

mamba install richdem -c conda-forge

**Comment by scottyhq** [comment]
Thanks @whitelightning450 , this is a lot of content to figure out, which notebook do you plan to cover for the 1 hour tutorial? The data files are too large to include in this repository and will have to be removed from the PR:

```
45M     data
1.7G    Predictions
```
I copied those files to `~/shared/geosmart-hackweek-2023/snow-extrapolation/` which everyone on the CryoCloud jupyterhub can access.

**Comment by whitelightning450** [comment]
Hi Scott,

No worries, I will try get to the event early and we can discuss. This is my first time completing a Jupyter book and it took me until Friday to get all of the content complete. As far as the content, I wanted to brush through the data processing as I feel that is very important for any ML project but wanted to focus on the model training/testing/validation/evaluation components. Rather than having a long single page, I broke them down into topics/chapters.

Hope to find you shortly and I can work on this during and after the event today to get it set up for my talk tomorrow. Worst case, I can run through it on Jupyter hub vs. the website.

Cheers,

Ryan Johnson, Ph.D.
AI Research Scientist, Alabama Water Institute
The University of Alabama<https://www.ua.edu/>
Cyber Hall ***@***.***,-87.5377403,18z/data=!4m8!1m2!2m1!1sCyber+Hall+1046+Box+870206+Tuscaloosa,+AL+35487!3m4!1s0x0:0xc959e6b07cd61c67!8m2!3d33.2195853!4d-87.5362462>
Box ***@***.***,-87.5377403,18z/data=!4m8!1m2!2m1!1sCyber+Hall+1046+Box+870206+Tuscaloosa,+AL+35487!3m4!1s0x0:0xc959e6b07cd61c67!8m2!3d33.2195853!4d-87.5362462>
Tuscaloosa, AL ***@***.***,-87.5377403,18z/data=!4m8!1m2!2m1!1sCyber+Hall+1046+Box+870206+Tuscaloosa,+AL+35487!3m4!1s0x0:0xc959e6b07cd61c67!8m2!3d33.2195853!4d-87.5362462>
***@***.******@***.***> | http://awi.ua.edu<http://awi.ua.edu/>
[The University of Alabama single line nameplate design without box A]<https://www.ua.edu/>

From: Scott Henderson ***@***.***>
Sent: Sunday, October 22, 2023 11:31 PM
To: geo-smart/2023-hackweek-website ***@***.***>
Cc: Ryan Johnson ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [geo-smart/2023-hackweek-website] added snow-extrapolation materials (PR #56)


Thanks @whitelightning450<https://github.com/whitelightning450> , this is a lot of content to figure out, which notebook do you plan to cover for the 1 hour tutorial? The data files are too large to include in this repository and will have to be removed from the PR:

45M     data

1.7G    Predictions

I copied those files to ~/shared/geosmart-hackweek-2023/snow-extrapolation/ which everyone on the CryoCloud jupyterhub can access.

-

**Comment by whitelightning450** [comment]
adjusted %pip installs

**Comment by whitelightning450** [comment]
Rebased to address _book/toc.yml problem

**Review by scottyhq — APPROVED** [review]
Thanks @whitelightning450 - Note lots of spelling errors (https://github.com/geo-smart/2023-hackweek-website/actions/runs/6623225516/job/17989919866?pr=56) you can follow up after your tutorial with final modifications to the content, but i'll go ahead and merge for tomorrow.

File: book/_config.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ parse:
 # See https://jupyterbook.org/content/execute.html
 execute:
   # NOTE: 'cache' to use jupyter-cache, or 'force' to run all
-  execute_notebooks: 'force'
+  execute_notebooks: 'off'
```
Comment (scottyhq) [inline book/_config.yml:38]: just turning off execution, so trusting things work as is on cryocloud jupyterhub. If you want to commit *rendered* notebooks with outputs we can try that in a new PR

**Pull request merged**